app/views/movies/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h2><%= @movie.title %></h2>
+<p><b>Studio: </b><%= @movie.studio.name %></p>
+<p><b>Creation Year: </b><%= @movie.creation_year %></p>
+<p><b>Genre: </b><%= @movie.genre %></p>
+<p><b>Actors: </b><br><% @movie.actors.sort_asc_age.each do |a|%>
+                       ID: <%= a.id %>  |  <%= a.name%> | Age: <%= a.age%><br>
+                       <% end%>
+
+<p><b>Average Age of Actors:</b> <%= @movie.average_age %> years old 
+
+<h3>Add an Existing Actor to Movie by ID</h3>
+  <%= form_with url: "/movies/#{@movie.id}", method: :get, data: { turbo: false } do |form| %>
+    <%= form.hidden_field :search, value: "actor" %>
+    <%= form.label :actor_id, "Add Existing Actor" %>
+    <%= form.text_field :actor_id %>
+    <%= form.submit "Submit" %>
+  <% end %>
+
+  <% if !@search_actors.nil? && @search_actors.any? %>
+  <% @search_actors.each do |actor| %>
+  <p><b>ID: <%= actor.id %>  |  <%= actor.name%> | Age: <%= actor.age%><br>
+  <%= button_to "Add ##{actor.id}", {controller: "movie_actor", action: "create", movie_id: @movie.id, actor_id: actor.id}, method: :post%>
```

Comment:
This works, but the approach is a little bit, uh, "extra" from what the user story is asking for ,which is to enter an existing Actor's ID and add them to the join table. 
That having been said, this is implemented well! Normally we don't need this double-layer approach though.